bridges/snowbridge/pallets/system-v2/src/lib.rs

```diff
@@ -222,6 +229,22 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::call_index(5)]
+		#[pallet::weight(<T as pallet::Config>::WeightInfo::add_tip())]
+		pub fn add_tip(
+			origin: OriginFor<T>,
```

Comment:
Should also forward the account of the user who initiated the tipping action. Will be useful if we need to track tips on BH that for some reason could not be applied.

```suggestion
			origin: OriginFor<T>,
			sender: T::AccountId
```